setup.py

```diff
@@ -79,7 +68,7 @@ def main(argv: List[str]) -> None:
         reqs = f.read()
         install_requires = reqs.strip().split("\n")
 
-    version = args.version
+    version = os.getenv("BUILD_VERSION")
```

Comment:
Our version on main is not updated to 0.4.0 I think. Is that going to be a problem?

https://github.com/pytorch/torchrec/blob/main/version.py

Reply:
Feel free to updated it to the desired version. I've noticed in the past that `torchrec.__version__` hasn't worked correctly, so we may want to mimic what other domain repos like vision do for versioning: https://github.com/pytorch/vision/blob/main/setup.py.